src/Couchbase.Lite.Shared/Serialization/FleeceMutableArray.cs

```diff
@@ -227,6 +255,44 @@ public bool Remove(object item)
 
         #endregion
 
+        #region IList<object>
+
+        public int IndexOf(object item)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void Insert(int index, object val)
```

Comment:
This whole method looks like it is copied from `Set`, so it would be good to find a way to call it.  It looks like it is simply just `Set` with the addition of `Native.FLMutableArray_Insert(ar, (uint)index, 1);`